docs/contributing/general.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: General
+category: Contributing
+parent:
+  label: All Docs
+  link: /
+---
+
+# General
+
+You can find the repository for this website on [GitHub](https://github.com/FuelLabs/docs-hub/tree/master). For instructions on how to run the repository locally, see the [README](https://github.com/FuelLabs/docs-hub/blob/master/README.md).
+
+## Contribution flow
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a git branch from where you want to base your work. This is usually master.
```

Comment:
This means they should create a feature branch, right? Should it say something like:

```suggestion
- Create a feature branch off of the master branch, which is typically the base for your work.
```